.travis.yml

```diff
@@ -1,14 +1,25 @@
 sudo: required
 dist: trusty
-
 language: cpp
-
 os:
- - linux
- - osx
-
+- linux
+- osx
 script:
-- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then bash ./verify-commit-messages.sh "$TRAVIS_COMMIT_RANGE" ; fi
+- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then bash ./verify-commit-messages.sh "$TRAVIS_COMMIT_RANGE"; fi
 - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then bash ./.travis/build-ubuntu_14_04.sh; fi
 - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then bash ./osx/qTox-Mac-Deployer-ULTIMATE.sh -i ; fi
 - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then bash ./osx/qTox-Mac-Deployer-ULTIMATE.sh -b ; fi
+# before_deploy:
+# - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then bash ./osx/qTox-Mac-Deployer-ULTIMATE.sh -d ; fi
+# - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then bash ./osx/qTox-Mac-Deployer-ULTIMATE.sh -dmg ; fi
+# deploy:
+#   provider: releases
+#   api_key:
+#     secure: Ji05dTgtKeOirakH6Y4N79Zm6eAeD3rAv0xNqis/mljSi6PxsVn2CqVJyqpHhy4yw6vutjFgxEBbLsaNFv6TcPli3lj1HsrUssYGsKPz0VTl4kxRPCiP+NjsBiuU4/tzb8kSyDkIA/spPwwmEWCEID8BtIDESIcQWMpInfqfMh/yxcK31e2X5AxWbWb90D9smS5B+Yee16KiDCzmc81vHO6k6TpSFxQifrUp5iS6xCLNxqGs6LUv6zBHVLmc2FxT4uNqXQewfV0fg7oW0ghTFo/k8CzfzPx0PHFzZcdmwDFXOX06cbCe1SAnvJnvrBFSnF2qrDMmATMZjGQlDEvWrWhEOZwYV4ntFcvnVTpuI+S+Q6QjREeJkwos2lAJ4bCPJOg+p+aEE0plek4X9JpJ3FRBW8bREwSVfhW+4wrm/uzd+nnLvO5QifZMpPBD9ptuQEJ1UeOQtAjjelnz0em3CpRZghPNsnq8fJBItmfcS6jDj/D1UHvwoHzewUj+L5ZI0wr7jZgkE27DfFVavDHiEir1EJyljY6CFC5pLdncx03lGba81VhBKfpGrpi0GEEmjJZaEiWO1MHoZAYl3dDeWl52Hj01GtTVcCJ/h1GKo12GqM89+cjPpetDaBkcQBLg6n60hc0S9wAsfdXdqgOrezkeznWtnE2uScc+mQYqC5I=
```

Comment:
I never tested to see what might happen if this section was un-commented but I just didn't want to risk possibly breaking things by doing so. The one change I might do though is to remove the trailing spaces that exist after `#` since travis is as picky about spaces as Python can be so only the `#` needs to be removed.